lib/gitx/cli/release_command.rb

```diff
@@ -12,7 +12,7 @@ class ReleaseCommand < BaseCommand
       include Gitx::Github
 
       desc 'release', 'release the current branch to production'
-      method_option :cleanup, :type => :boolean, :desc => 'cleanup merged branches after release'
+      method_option :cleanup, type: :boolean, desc: 'cleanup merged branches after release'
```

Comment:
Line is too long. [91/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.